### ISSUE TITLE
Help!  Stuck on "Dumping NOR 1/8"

### ISSUE THREAD

**Issue by thesupernoah64** [axi0mX/ipwndfu#14]
Hi, I have executed the command to run the alloc8 patcher (./ipwndfu -x) and am stuck on "Dumping NOR part 1/8".  Is there something wrong?  I tried 2 versions of iTunes (iTunes 10.6.3 and 11.0.3) and no solution.  PyUSB, Libusb, and the iBSS files are installed correctly.  I have Python 2.7.  I am also running OS X Snow Leopard 10.6.8.  Does this exploit even work on Snow Leopard?

Mac:  iMac Intel Core 2 Duo Late 2006 1.83 GHZ

Note that I am NOT running this on a virtual machine.

**Comment by axi0mX** [comment]
It should work. Could you try again? Hard reboot the phone (hold Power + Home button for 5-6 seconds), then run ```./ipwndfu -p``` and ```./ipwndfu -x``` and let me know if this happens again.

You did not patch libusb, correct? Libusb should only be patched on El Capitan and Sierra.

**Comment by thesupernoah64** [comment]
Nope I've never patched libusb.

**Comment by thesupernoah64** [comment]
I used Pwnage tool 3.1.5 to make my custom iOS 3.1.3 firmware, and my iPhone 3GS should be compatible as I got no Error 28 in iTunes.

**Comment by axi0mX** [comment]
Cool, you're on the right track then. Did you have a chance to try it again?

**Comment by thesupernoah64** [comment]
I tried again, same thing.  Still stuck on Dumping NOR part 1/8.

**Comment by axi0mX** [comment]
Could you unplug the cable, plug it back in, and try again without rebooting the phone?

**Comment by thesupernoah64** [comment]
Sure

**Comment by thesupernoah64** [comment]
Unplugged it, tried again, still stuck.  Do you think this has something to do with the firmware I restored to?

**Comment by axi0mX** [comment]
No, I believe you did everything correctly. I don't have other things for you to try for now. If you have a newer Mac to test it on, Mavericks or Yosemite might work better and won't require patching libusb.

I did not test this on Snow Leopard yet. I'll see if I can get Snow Leopard installed on my 2011 MacBook Air. It seems that it might be possible: https://forums.macrumors.com/threads/snow-leopard-on-mba-2011-fully-working.1352079/

**Comment by MorpheinD** [comment]
I think I might have patched libusb but it was working earlier until getting stuck at the ibss file saying its corrupt but i figured how to get the correct file 
when I ran the code you had listed on the main ipwndfu page & the file left in my home folder was blank i.e. get info said it had 0kb of info stored in the file so was I opened the 4.3.5 ipsw with archive utility and then in terminal i ran running unzip -p iBSS.n88ap.RELEASE.dfu > n88ap-iBSS-4.3.5.img3 (also I dragged and dropped the ibss file in the dfu folder) and BLAM! that was the correct file so I moved it to the ipwndfu-master folder and now I am stuck at the same place & it doesnt matter what I do ive even re-loaded the custom ipsw thru itunes idk what to do if I did happen to patch libusb by accident 1. how can I tell amd 2. how can I fix it ? PLEASE HELP

**Comment by thesupernoah64** [comment]
I got it working now!  I tried again on Mavericks and worked.  I don't think this is compatible with Snow Leopard.

**Comment by 128keaton** [comment]
@thesupernoah64 it literally might be because of the version of Python. I have a Snow Leopard machine to try let me give that a spin
